packages/app/src/App.tsx

```diff
@@ -1,29 +1,29 @@
 import { FC } from "react";
 import { Route, Switch } from "react-router";
+import { Box, Grid } from "grommet";
 import { BrowserRouter as Router } from "react-router-dom";
 
-import TestnetBanner from "./components/testnet-banner";
 import Home from "./pages/home";
 import Token from "./pages/token";
 import { NotificationsList } from "./components/transactions";
 import Foot from "./components/footer";
-import { Grid } from "grommet";
+import Navbar from "./components/nav";
 
 const App: FC = () => {
   return (
-    <>
-      <Grid fill rows={["xxsmall", "flex", "xsmall"]}>
-        <TestnetBanner />
-        <Router>
+    <Router>
+      <Grid height="100vh" rows={["xsmall", "1fr"]}>
```

Comment:
The grid can be larger than 100vh

Reply:
changed it to `min-height` to be more explicit, but it did have the same behavior (without it it does not fill the screen)

Reply:
Right, but we also want to add additional content to the home page below the initial pagefold eventually so being more explicit here doesn't hurt I guess